benchopt/tests/test_benchmarks/dummy_benchmark/solvers/sklearn.py

```diff
@@ -22,8 +23,11 @@ def set_objective(self, X, y, lmbd):
         warnings.filterwarnings('ignore', category=ConvergenceWarning)
 
     def run(self, n_iter):
-        self.clf.max_iter = n_iter
-        self.clf.fit(self.X, self.y)
+        if n_iter == 0:
+            self.coef = np.zeros([self.X.shape[1] + self.fit_intercept])
+        else:
+            self.clf.max_iter = n_iter
+            self.clf.fit(self.X, self.y)
```

Comment:
This change is unrelated to the PR so avoid adding it in the PR in the future.

Reply:
You could have keep it @Melvin-klein :)
I said this because I did not see that it was @agramfort  who asked you to do this here, else I would have complain directly to him ;)